src/FragmentCache.php

```diff
@@ -160,28 +159,14 @@ public function calculateKey()
         return array_merge([__CLASS__, $this->id], (array) $this->variations);
     }
 
-    /**
-     * {@see $cache}
-     *
-     * @param CacheInterface|null $value
-     *
-     * @return FragmentCache
-     */
-    public function cache(?CacheInterface $value): FragmentCache
-    {
-        $this->cache = $value;
-
-        return $this;
-    }
-
     /**
      * {@see $content}
      *
      * @param string|bool $value
      *
-     * @return FragmentCache
+     * @return $this
      */
-    public function content($value): FragmentCache
+    public function content($value): self
     {
         $this->content = $value;
```

Comment:
`$value` is declared as `string|bool` but it seems that's not correct according to property declaration.

Reply:
fixed.